extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -70,5 +80,21 @@
 		createCookie('qualaroo_survey_submission');
 	});
 
+	if (window.ads.context.opts.sourcePointDetection) {
+		if (!window.ads.runtime || !window.ads.runtime.sp || window.ads.runtime.sp.blocking === undefined) {
```

Comment:
Actually it's not the same; firstly we check if sourcePointDetection is enabled not sourcePointRecovery, secondly we check if blocking is undefined (so not set to false or true).